index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>CV</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>Vladislav Lisetskii</h1>
+    <hr>
+    <div class="info">
+      <div class="personal-info">
+        <ul>
+          <li>Position: Fullstack developer (Java/Javascript)</li>
+          <li>Age: 25</li>
+          <li>City: Moscow</li>
+        </ul>
+      </div>
+      <div class="contacts">
+        <ul>
+          <li>E-mail: <a href="mailto:vladlis54@gmail.com">vladlis54@gmail.com</a></li>
+          <li>Phone: <a href="phoneto:88005553535">8 (800) 555-35-35</a></li>
+          <li>Github: <a href="https://github.com/Vladlis">Vladlis</a></li>
+        </ul>
+      </div>
+    </div>
+    <img src="assets/photo.jpeg" alt="photo" class="photo">
+    <hr>
+  </header>
+  <section class="summary">
+    <h2>Summary</h2>
+    <span>I am a Java developer who became tired of constant enterprise development hell, so I have mastered frontend
```

Comment:
apparently that should be a paragraph
use span when there is no more suitable tag for some inline element
use div when there is no better tag for a block element
and when you have text in a block element it is a paragraph

Reply:
thanks